vendor/k8s.io/kubernetes/staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -522,9 +522,7 @@ func (g *Cloud) ensureInternalInstanceGroups(name string, nodes []*v1.Node) ([]s
 				skip.Insert(groupInstances.UnsortedList()...)
 			}
 		}
-		if remaining := names.Difference(skip).UnsortedList(); len(remaining) > 0 {
-			gceZonedNodes[zone] = remaining
-		}
+		gceZonedNodes[zone] = names.Difference(skip).UnsortedList()
```

Comment:
This nuked the changes carry from https://github.com/openshift/origin/pull/24135****